ui/jstests/test-learning-resource.jsx

```diff
@@ -176,17 +176,128 @@ define(['QUnit', 'jquery', 'react', 'lodash', 'learning_resources',
     function (assert) {
       var done = assert.async();
 
+      var vocab = $.extend({}, selectedVocabulary);
+      var appendTermSelectedVocabulary = function(term) {
+        vocab.terms = vocab.terms.concat(term);
+      };
+
+      var loadedState;
+      var setLoadedState = function(loaded) {
```

Comment:
I assume this function expects a boolean? Out of curiosity, do we have any conventions about parameter naming to help with clarity? Maybe just a comment would be helpful.